pkg/providers/instance/instance.go

```diff
@@ -67,6 +67,7 @@ var (
 	SKUNotAvailableReason                       = "SKUNotAvailable"
 
 	SubscriptionQuotaReachedTTL = 1 * time.Hour
+	AllocationFailureTTL        = 15 * time.Minute
```

Comment:
15 minutes based on the behavior I saw when investigating the ZonalAllocationFailures - after around 15~ minutes, node claims were getting deleted by the controller, the goal is to block SKUs that failed with ZonalAllocation for long enough time that more different SKUs are tried before node claim is deleted.

Reply:
Adjusted to 1 hr based on offline discussion